ocaml/xapi/xapi_sr.ml

```diff
@@ -188,6 +188,53 @@ let probe ~__context ~host ~device_config ~_type ~sm_config =
        | Probe _ as x -> Xmlrpc.to_string (rpc_of_probe_result x)
     )
 
+let probe_ext ~__context ~host ~device_config ~_type ~sm_config =
```

Comment:
Maybe we should make probe call probe_ext and transform it into the old format, Jon recommended something like that.